vector/runner.go

```diff
@@ -566,6 +567,12 @@ func modelTypeFromTable(tableName string, expected json.RawMessage, actual []int
 			}
 			actType = append(actType, act)
 		}
+		sort.Slice(actType, func(i, j int) bool {
```

Comment:
Ensures we compare results in the correct order while running test vectors. The new H/AMT diffing logic returns elements in a different order than the generic diffing implementation.